src/js/ko-components/templates/orifice-calculator.haml

```diff
@@ -5,7 +5,7 @@
       %select{ :id => 'pipeID', "data-bind" => "options: pipeID, value: selectedPipeID" }
     %li
       %label Operating Pressure
-      %input{ :type => 'text', "data-bind" => "value: operatingPressure" }
+      %input{ :id => 'operatingPressure', :type => 'text', "data-bind" => "value: operatingPressure" }
```

Comment:
`%input#operating-pressure{ :type => 'text', "data-bind" => "value: operatingPressure" }`

html/haml id's use dashes and not camel case